src/microsoft-trydotnet-editor/package.json

```diff
@@ -50,13 +50,14 @@
     "ts-mocha": "10.0.0",
     "ts-node": "10.8.1",
     "typescript": "4.7.4",
-    "webpack": "5.73.0",
+    "webpack": "^5.90.3",
     "webpack-cli": "4.10.0"
   },
   "dependencies": {
-    "@microsoft/polyglot-notebooks": "1.0.441801",
+    "@microsoft/polyglot-notebooks": "^1.0.515601",
```

Comment:
Similar comment about versioning here as well.

Reply:
reverted this change